FreeRTOS/FreeRTOS
# Add WinPCap NetworkInterface Changes

Description
-----------
Add changes to support backward compatibility for xWinPcap_NetworkInterfaceInitialise.

Test Steps
-----------
Demo build compilation.



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
